src/main/scala/fs2/async/immutable/Signal.scala

```diff
@@ -0,0 +1,102 @@
+package fs2.async.immutable
+
+import fs2.{Async, process1, Stream}
+import fs2.util.Functor
+
+import fs2.async.{immutable, mutable}
+
+
+/**
+ * Created by pach on 10/10/15.
+ */
+trait Signal[F[_],A]  {
+  /**
+   * Returns a continuous stream, indicating whether the value has changed.
+   * This will spike `true` once for each time the value of `Signal` was changed.
+   * It will always start with `true` when the process is run or when the `Signal` is
+   * set for the first time.
+   */
+  def changed: fs2.Stream[F, Boolean]
```

Comment:
can you get rid of the `fs2.` prefix everywhere? It is not needed if you've imported `fs2.Stream`. If you need to reference `scala.collection.immutable.Stream`, just refer to it qualified or import it with a different name.